database/factories/ModelFactory.php

```diff
@@ -12,13 +12,131 @@
 */
 
 /** @var \Illuminate\Database\Eloquent\Factory $factory */
-$factory->define(App\User::class, function (Faker\Generator $faker) {
+$factory->define(App\Model\User::class, function (Faker\Generator $faker) {
     static $password;
 
     return [
-        'name' => $faker->name,
-        'email' => $faker->unique()->safeEmail,
+        'username' => $faker->unique()->userName,
         'password' => $password ?: $password = bcrypt('secret'),
-        'remember_token' => str_random(10),
+        'email' => $faker->unique()->safeEmail,
+        'full_name' => $faker->name,
+        'phone' => $faker->phoneNumber,
+        'is_admin' => $faker->boolean(),
+        'is_active' => $faker->boolean(),
+    ];
+});
+
+$factory->define(App\Model\Guest::class, function (Faker\Generator $faker) {
+
+    return [
+        'full_name' => $faker->name,
+        'email' => $faker->unique()->safeEmail,
+        'phone' => $faker->phoneNumber,
+    ];
+});
+
+$factory->define(App\Model\Place::class, function (Faker\Generator $faker) {
+
+    return [
+        'name' => $faker->city,
+        'descript' => $faker->text,
+    ];
+});
+
+$factory->define(App\Model\Service::class, function (Faker\Generator $faker) {
+    
+    return [
+        'name' => $faker->word,
+    ];
+});
+
+$factory->define(App\Model\Hotel::class, function (Faker\Generator $faker) {
+
+    return [
+        'name' => $faker->company,
+        'address' => $faker->address,
+        'star' => $faker->numberBetween($min = 1, $max = 5),
+        'place_id' => App\Model\Place::all()->random()->id,
```

Comment:
Nếu mình gọi cái ni: factory(App\Model\Hotel::class, 15)->create();
Thì bị lặp query này: `App\Model\Place::all()->random()->id` 15 lần.

Xem có cách nào mình truyền cái ni qua paramter không?
Tham khảo cái ni: https://stackoverflow.com/questions/32378215/how-to-pass-arguments-to-laravel-factories